tests/test_samples.rs

```diff
@@ -224,6 +225,13 @@ fn all_the_fields() {
     assert_eq!(path_dep.source, None);
     assert_eq!(path_dep.kind, DependencyKind::Normal);
     assert_eq!(path_dep.req, semver::VersionReq::parse("*").unwrap());
+    assert_eq!(
+        path_dep
+            .path
+            .as_deref()
```

Comment:
Please don't use `as_deref`, otherwise we need to bump MSRV

Reply:
This is in the tests though, not in the main library? Will change anyway, but I wonder if we should exempt tests from being subject to MSRV.